src/MCT.js

```diff
@@ -325,6 +325,24 @@ define([
         });
     };
 
+    /**
+     * Get path to where assets are hosted.
+     * @memberof module:openmct.MCT#
+     * @method getAssetPath
+     */
+    MCT.prototype.getAssetPath = function () {
+        var constants = this.legacyBundle.extensions.constants;
```

Comment:
Let's not use Angular for this, we don't need it. Let's just store it on a local variable.

eg.
``` JavaScript
MCT.prototype.setAssetPath = function (assetPath) {
     this._assetPath = assetPath;
}
MCT.prototype.getAssetPath = function () {
     return this._assetPath || '/';
}
```

Reply:
Ahh this is Angular stuff you were talking about. Let me change it.